### PR TITLE
docs: reference .NET library for backend development

### PR DESCRIPTION
**Pull request by takla21**
GitHub Issue: #

## Proposed Changes

 - [ ] Bug fix
 - [ ] Feature
 - [ ] Code style update (formatting)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build or CI related changes
 - [x] Documentation content changes
 - [ ] Other, please describe:

## Description

I've added a reference to the .NET library for backend development

## Impact on version

- [ ] **Major** (Public API was modified.)
  - Public constructs (class, struct, delegate, enum, etc.) were removed or renamed.
  - Public members were removed or renamed.
  - Public method signatures were changed or renamed.
- [ ] **Minor** (Public API was extended.)
  - Public constructs, members, or overloads were added.
- [x] **Patch** (Public API was unchanged.)
  - A bug in behavior was fixed.
  - Documentation was changed.
- [ ] **None** (The library is unchanged.)
  - Only code under the `build` folder was changed.
  - Only code under the `.github` folder was changed.

## PR Checklist 

- [x] Your conventional commits are aligned with the **Impact on version** section.
- [x] Documentation is up to date.
  - Content of `README.md` is up to date.
  - XML documentation is up to date.
- [x] The [BREAKING_CHANGES.md](../BREAKING_CHANGES.md) document is up to date.
  - Create a new Major.0.0 header if you do a **Major** change and list the breaking changes.
- [x] Tested on all relevant platforms

### CODE REVIEW

**Review by Lee31416 — APPROVED** [review]

**Review by Marc-Antoine-Soucy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,7 +4,9 @@
 
 Signs and validates HTTP requests.
 
-This projects can help you implements [HMAC](https://en.wikipedia.org/wiki/HMAC) signature to HTTP requests on flutter projects.
+This projects can help you implements [HMAC](https://en.wikipedia.org/wiki/HMAC) signatures to HTTP requests on flutter projects.
+
+For the backend part, use the [RequestsSignature](https://github.com/nventive/RequestsSignature) library to implement [HMAC](https://en.wikipedia.org/wiki/HMAC) signatures.
```
Comment (Soap-141) [inline README.md]: Backend can be in many things and not only C#/.NET, should we specify?

**Review by Soap-141 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jeanplevesque) [reply README.md]: This recipe currently only exists in .NET and Dart.

Comment (Soap-141) [reply README.md]: But someone could use this package for his flutter application and have a rust backend, so he'll have to do the implementation himself. Like we could say that for the .NET BE part use this?

File: README.md [hunk 2]
```diff
@@ -20,7 +22,7 @@ This will add a line like this to your package's `pubspec.yaml` (and run an impl
 
 ```yaml
 dependencies:
-  request_signature_dart: ^0.0.1
+  request_signature_dart: ^1.0.0
```
Comment (jeanplevesque) [inline README.md]: it's request**s**_signature_dart

File: README.md (hunk 1 above)
Comment (jeanplevesque) [reply README.md]: yeah we could specify that it's .NET

File: README.md (hunk 2 above)
Comment (takla21) [reply README.md]: oups :S

File: README.md (hunk 1 above)
Comment (takla21) [reply README.md]: I've reworded it to specify backend in .NET

**Pull request merged**